### PR TITLE
chore: update README.md for command-response bot

### PR DESCRIPTION
**Pull request by dooriya**

### CODE REVIEW

File: templates/bot/js/command-and-response/README.md [hunk 1]
```diff
@@ -1,3 +1,86 @@
...
+| `bot` | The bot source code |
+| `templates` |Templates for Teams app manifest and corresponding Azure resources|
+
+The core command-response implementation is in `bot/` folder, containing following content:
+
+| File / Folder | Contents |
+| - | - |
+| `src/adaptiveCards/` | Adaptive card templates |
+| `src/internal/initialize.ts(js)` | Generated initialize code for initialize the command bot |
+| `src/helloworldCommandHandler.ts` | A hello world command handler to process a helloworld command and return an adaptive card as response |
```
Comment (swatDong) [inline templates/bot/js/command-and-response/README.md:31]: ts(js)

File: templates/bot/js/command-and-response/README.md [hunk 2]
```diff
@@ -1,3 +1,86 @@
...
+| `src/index.ts(js)` | The entrypoint to handle bot messages and send response |
+| `.gitignore` | The git ignore file to exclude local files from bot project |
+| `package.json` | The NPM package file for bot project |
+
+### Bot Initialization
+
+The default initialization is located in `bot/src/internal/initialize.ts(js)`, which creates a default [Bot Freamework adapter](https://docs.microsoft.com/en-us/javascript/api/botbuilder/botframeworkadapter?view=botbuilder-ts-latest) and sets up the TeamsFx command bot. You can also add your own initialization logic here to:
```
Comment (swatDong) [inline templates/bot/js/command-and-response/README.md:38]: :P 
Just my personal habit - I'll remove all {locale} part (e.g., /en-us/) from docs link~

**Review by Siglud — APPROVED** [review]

File: templates/bot/js/command-and-response/README.md (hunk 2 above)
Comment (dooriya) [reply templates/bot/js/command-and-response/README.md:38]: Good practice for globalization~

File: templates/bot/js/command-and-response/README.md (hunk 1 above)
Comment (dooriya) [reply templates/bot/js/command-and-response/README.md]: Fixed. Just use `.ts` for TypeScript readme, and `.js` for JavaScript readme.

**Review by Siglud — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Pull request merged**